ibm/service/power/data_source_ibm_pi_instances.go

```diff
@@ -170,59 +192,57 @@ func dataSourceIBMPIInstancesAllRead(ctx context.Context, d *schema.ResourceData
 
 	var clientgenU, _ = uuid.GenerateUUID()
 	d.SetId(clientgenU)
-	d.Set("pvm_instances", flattenPvmInstances(powervmdata.PvmInstances))
+	d.Set(Attr_PVMInstances, flattenPvmInstances(powervmdata.PvmInstances))
 
 	return nil
 }
 
 func flattenPvmInstances(list []*models.PVMInstanceReference) []map[string]interface{} {
 	result := make([]map[string]interface{}, 0, len(list))
 	for _, i := range list {
-
 		l := map[string]interface{}{
-			"pvm_instance_id":             *i.PvmInstanceID,
-			"memory":                      *i.Memory,
-			"processors":                  *i.Processors,
-			"proctype":                    *i.ProcType,
-			"status":                      *i.Status,
-			"minproc":                     i.Minproc,
-			"minmem":                      i.Minmem,
-			"maxproc":                     i.Maxproc,
-			"maxmem":                      i.Maxmem,
-			"pin_policy":                  i.PinPolicy,
-			"virtual_cores_assigned":      i.VirtualCores.Assigned,
-			"max_virtual_cores":           i.VirtualCores.Max,
-			"min_virtual_cores":           i.VirtualCores.Min,
-			"storage_type":                i.StorageType,
-			"storage_pool":                i.StoragePool,
-			"storage_pool_affinity":       i.StoragePoolAffinity,
-			"license_repository_capacity": i.LicenseRepositoryCapacity,
-			PIPlacementGroupID:            i.PlacementGroup,
-			"networks":                    flattenPvmInstanceNetworks(i.Networks),
+			Attr_LicenseRepositoryCapacity: i.LicenseRepositoryCapacity,
+			Attr_MaxMem:                    i.Maxmem,
+			Attr_MaxProc:                   i.Maxproc,
+			Attr_MaxVirtualCores:           i.VirtualCores.Max,
+			Attr_Memory:                    *i.Memory,
+			Attr_MinMem:                    i.Minmem,
+			Attr_MinProc:                   i.Minproc,
+			Attr_MinVirtualCores:           i.VirtualCores.Min,
+			Attr_Networks:                  flattenPvmInstanceNetworks(i.Networks),
+			Attr_PinPolicy:                 i.PinPolicy,
+			Attr_PlacementGroupID:          i.PlacementGroup,
+			Attr_Processors:                *i.Processors,
+			Attr_ProcType:                  *i.ProcType,
+			Attr_PVMInstanceID:             *i.PvmInstanceID,
+			Attr_Status:                    *i.Status,
+			Attr_StoragePool:               i.StoragePool,
+			Attr_StoragePoolAffinity:       i.StoragePoolAffinity,
+			Attr_StorageType:               i.StorageType,
+			Attr_VirtualCoresAssigned:      i.VirtualCores.Assigned,
 		}
 
 		if i.Health != nil {
-			l["health_status"] = i.Health.Status
+			l[Attr_HealthStatus] = i.Health.Status
 		}
 
 		result = append(result, l)
-
 	}
 	return result
 }
 
+// TODO: REVIEW THIS ONE
```

Comment:
I think this is the only thing for this PR

Reply:
Changes made